redash/authentication/remote_user_auth.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from flask import redirect, url_for, Blueprint, request
+from redash.authentication.google_oauth import create_and_login_user
+from redash.authentication.org_resolving import current_org
+from redash import settings
+
+logger = logging.getLogger('remote_user_auth')
+
+blueprint = Blueprint('remote_user_auth', __name__)
+
+@blueprint.route("/remote_user/login")
+def login():
+    next_path = request.args.get('next')
+
+    if not settings.REMOTE_USER_LOGIN_ENABLED:
+        logger.error("Cannot use REMOTE_USER for login without being enabled in settings")
+        return redirect(url_for('index', next=next_path))
+
+    email = request.environ.get("REMOTE_USER", request.headers.get(settings.REMOTE_USER_HEADER))
```

Comment:
No, I'm using a header.  I added it for completeness.  I imagine someone could be using mod_wsgi, or another server instead of gunicorn, that would provide `REMOTE_USER` through the environment, although I realize you may not be interested in supporting that setup.  I don't have a vested interest in keeping the env check if you want to remove it.